### DISCUSSION TITLE
加载长图报错Failed to instantiate texture from proxy.

### DISCUSSION THREAD

**Discussion by l540144279** [Tencent/libpag#1580] [Q&A]
RT，加载渲染一张长图，宽高尺寸630*10000，渲染时报错Failed to instantiate texture from proxy. 求解，是底层限制了图片最大尺寸吗？或是纹理渲染有最大限制，版本4.1.49
![image](https://github.com/Tencent/libpag/assets/12799090/66b0ac8f-3d46-46ae-9f00-26240ed50d22)

**Comment by kevingpqi123** [comment]
原因有多种，最大尺寸好像没有限制，如果是必现的，接入源码加个断点就定位到原因了

**Comment by kevingpqi123** [comment]
一周之内没回复，暂时关闭

**Comment by myongong** [comment]
我也出现了这个问题，同一个资源某些的手机报错
我把官方演示版本运行播放，也会报错。但是如果先播放了其他PAG文件，再去播放有问题的资源，又可以播。

官方演示Demo部分打印报错
HardwareDecoder:decoder session create failed status = -12911
VideoReader::makeVideoDecoder failure, reset factoryIndex
2024-01-08 20:52:37.514108+0800 pag-ios[517:69580] Metal API Validation Enabled
Failed to instantiate texture from proxy.
HardwareDecoder:decoder session create failed status = -12911
VideoReader::makeVideoDecoder failure, reset factoryIndex
HardwareDecoder:decoder session create failed status = -12911
VideoReader::makeVideoDecoder failure, reset factoryIndex
Failed to instantiate texture from proxy.

我尝试了软解，也报错
VideoReader::makeVideoDecoder failure, reset factoryIndex
2024-01-08 20:55:40.090911+0800 pag-ios[522:71745] Metal API Validation Enabled
Failed to instantiate texture from proxy.
VideoReader::makeVideoDecoder failure, reset factoryIndex
VideoReader::makeVideoDecoder failure, reset factoryIndex
Failed to instantiate texture from proxy.
VideoReader::makeVideoDecoder failure, reset factoryIndex
Failed to instantiate texture from proxy.

**Reply by myongong** [reply]
PS 我这个资源很小
